boards/pic32-wifire/doc.txt

```diff
@@ -3,19 +3,88 @@
 @ingroup     boards
 @brief       Support for the Digilent PIC32 WiFire
 
-For instructions on reflashing see:
+## Overview
 
-https://docs.creatordev.io/wifire/guides/wifire-programming/
+The ChipKit Wifire is a development board by Digilent featuring a PIC32MZ and a Wifi module (MRF24WG0MA).
+General information about the board can be found here: https://store.digilentinc.com/wi-fire-wifi-enabled-pic32mz-microcontroller-board/
+Please do not use board rev A or B as they use a different MCU (PIC32MZ2048ECG100).
 
-The RIOT build generates a hexfile compatible with MPLAB-IPE.
+Schematics for the board (rev C): https://reference.digilentinc.com/_media/reference/microprocessor/wi-fire/chipkit_wifire_sch.pdf
```

Comment:
use bullet list for both schematics and also Markdown links [rev C](<url>) ... [rev D](<url>)

Reply:
Fixed